### PR TITLE
Install README and NEWS files

### PR DESCRIPTION
**Pull request by graahnul-grom**
- Copy `README.md` and `NEWS.md` files to `$(docdir)`
(`share/doc/lepton-eda/`) on "make install".
- Fix `uninstall-` target:
  - respect `$(DESTDIR)`
  - remove $(pkgdatadir) (`share/lepton-eda/`) on "make uninstall"

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -83,15 +83,16 @@ precompile:
 	$(bindir)/lepton-schematic
 
 
-uninstall-local:
-	rm -fr $(datadir)/lepton-eda/ccache
```
Comment (vzh) [inline Makefile.am]: Is this change intended? I'm sorry for my ignorance wrt what the code used for.

Comment (graahnul-grom) [reply Makefile.am]: this is superseded by `rm -fr $(pkgdatadir)`
`datadir = share`
`pkgdatadir = share/lepton-eda`

Comment (vzh) [reply Makefile.am]: OK.

**Comment by vzh** [comment]
Hope you'll update NEWS if there is need to.
